tests/Dockerfile.tests

```diff
@@ -1,5 +1,9 @@
-FROM centos:7
+FROM registry.access.redhat.com/ubi8/go-toolset:latest as builder
+COPY . .
+RUN CGO_ENABLED=0 GOOS=linux go test ./...
+RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo ./cmd/vault-manager
```

Comment:
why not making these commands `Makefile` targets?

Reply:
Good suggestion. Feel free to send a PR :)